simple_calculator/mystyle.css

```
@@ -1,3 +1,7 @@
+.cal-group-class {
```

`simple_calculator/index.html` 파일의 15번째줄에

```html
<div class="cal-group">
```

와 같이, `cal-group`이라는 이름의 className을 쓰고 있기 때문에, 여기 css에서도 `.cal-group`이라는 이름을 써야 매칭이 됩니다.

```suggestion
.cal-group {
```